### ISSUE TITLE
Why doesn't full-text search work? NOW: How to add a field

### ISSUE THREAD

**Issue by mroswell** [mroswell/list-N#21]

**Comment by mroswell** [comment]
https://docs.datasette.io/en/latest/full_text_search.html

**Comment by mroswell** [comment]
Seems to work when enabled as follows:
$ sqlite-utils enable-fts listN.db listN 'Active ingredients' 'Product name' Company 'Formulation type' 'Surface type' 'Use site' 'Why on List N' 'Follow directions for this virus'

**Comment by mroswell** [comment]
Hmm... want to add 'Registration Number' to full text search... but "sqlite3.OperationalError: table [listN_fts] already exists"

**Comment by mroswell** [comment]
1. Enable Full text search:
` $ sqlite-utils enable-fts listN.db listN 'Surface_type' 'Active_ingredient' 'Safer_or_Toxic' 'Date_on_List_N'  'Company' 'Contact_time' 'Use_site' 'Product_name'  'Active_ingredients' 'Formulation_type' 'Follow_directions_for_this_virus' 'Why_on_List_N' 'Registration_number'
`
2. I deleted and started again when I needed to add a missing column, but looks like this would also work (Then go back to something like like step 1.)
`$ sqlite-utils disable-fts listN.db documents`

3. Understand that full search doesn't do partial word search, or any sort of spell correction.
